src/gvars.c

```diff
@@ -1358,16 +1467,32 @@ static StructGVarFunc GVarFuncs [] = {
 static Int InitKernel (
     StructInitInfo *    module )
 {
-  ValGVars = (Obj) 0;
-  NCopyAndFopyGVars = 0;
-  NCopyAndFopyDone = 0;
```

Comment:
In case anybody wonders: The three variables above only were (re)initialized here for a long-ago abandoned project to make the GAP kernel "restartable" (@stevelinton added that in 2004, and I removed the incomplete code in 2014). So this basically reverts the code to how it was before 2004, and how most other kernel modules do it.